### PR TITLE
Add a base stats method

### PR DESCRIPTION
**Pull request by nkatzman**
Adds a dummy method that returns statistics about the executor

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -80,6 +80,15 @@ class Executor {
     _stream(config, callback) {
         callback(new Error('not implemented'));
     }
+
+    /**
+     * Return statistics on the executor
+     * @method status
```
Comment (tkyi) [inline index.js]: stats

**Comment by tkyi** [comment]
👍  LGTM

**Comment by petey** [comment]
👍

**Comment by tkyi** [comment]
👍 
🐑 it

**Pull request merged**